src/modals/cpu-pinning/CpuPinningPolicy.js

```diff
@@ -15,7 +15,11 @@ const CpuPinningPolicy = {
   },
 
   isDynamic (policy) {
-    return !this.isManual(policy) && !this.isNone(policy)
+    return policy && !this.isManual(policy) && !this.isNone(policy)
+  },
+
+  isExclusive (policy) {
+    return policy && this.isNone(policy)
```

Comment:
shouldn't we check that the policy is set to dedicated here?

Reply:
yes, you're right, this was implemented when the dedicated was not yet in the rest api

Reply:
I think you need to implement the 'follow' on the api side